src/systemctl/systemctl.c

```diff
@@ -1158,7 +1157,7 @@ static int run(int argc, char *argv[]) {
         if (r <= 0)
                 goto finish;
 
-        if (proc_mounted() == 0)
+        if (!arg_no_warn && proc_mounted() == 0)
                 log_warning("%s%s/proc/ is not mounted. This is not a supported mode of operation. Please fix\n"
```

Comment:
please print this warning, but downgrade from LOG_WARNING to LOG_DEBUG in case of arg_no_warn